platform/posix/transport/src/mbedtls_pkcs11_posix.c

```diff
@@ -85,8 +81,12 @@ static const char * pNoLowLevelMbedTlsCodeStr = "<No-Low-Level-Code>";
 
 /**
  * @brief Debug logging level to use for MbedTLS.
+ *
+ * @note The default value of 0 disables MbedTLS logging.
+ * See <https://tls.mbed.org/api/debug_8h.html#a6629362e96b43725ace95c8ff01d9985>
```

Comment:
Why wrap the URL in angle brackets?
```suggestion
 * See https://tls.mbed.org/api/debug_8h.html#a6629362e96b43725ace95c8ff01d9985
```